lib/beaker/dsl/structure.rb

```diff
@@ -54,6 +55,51 @@ def step step_name, &block
         end
       end
 
+      # Provides a method to help manual tests. So we can use beaker to set up
+      # the environment, then prompt a user to manually check the setup.
+      # @param [String] step_name The name of the step to be logged.
+      # @param [String] fail_message The message to return if this manual step failed
+      def manual_step step_name, fail_message='Manual step failed.'
+        logger.notify "\n* #{step_name}\n"
+        if(!@options.has_key?(:exec_manual_tests))
+          # if the option -exec-manual-tests is not set then this has executed outside of a manual tests
+          # so we raise an error to avoid issues
+          raise('--exec-manual-tests option not set, this means a manual_step was used outside a manual_test')
+        end
+
+        set_current_step_name(step_name)
+        # Here we prompt the user to tell us if the step passed or failed
+        loop do 
+          input = Readline.readline('did this step pass, Y/n?', true).squeeze(" ").strip
+          if input == 'Y'
+            break
+          elsif input == 'n'
+            raise Beaker::DSL::FailTest, fail_message
```

Comment:
Regarding pry; that would be a good subsequent update to this. It wouldn't be useful in all cases; but it would be a handy option for immediate investigation.
You can probably also mix this new option with the --debug-errors options and mark this test failed then immediately drop into pry.